_config.yml

```diff
@@ -83,13 +83,11 @@ collections:
     output: true
     order:
       - hello_world.md
-      - constants_and_variables.md
-      - use_functions.md
-      - comments.md
-      - control_flow.md
-      - structs.md
-      - enums.md
-      - typealiases.md
+      - declare_constants_or_variables.md
+      - comments_in_glu.md
+      - structs_in_glu.md
+      - operators_overload.md
+      - templating_structures.md
```

Comment:
that's no good

Reply:
the conflicts weren't resolved correctly, those articles were renamed and some are missing.